src/components/ui/coaching-session.tsx

```diff
@@ -15,56 +15,49 @@ import {
   DropdownMenuTrigger,
 } from "@/components/ui/dropdown-menu";
 import { MoreHorizontal } from "lucide-react";
-import { CoachingSessionDialog } from "@/components/ui/dashboard/coaching-session-dialog";
-import { DateTime } from "ts-luxon";
+import { CoachingSession as CoachingSessionType } from "@/types/coaching-session";
 
 interface CoachingSessionProps {
-  coachingSession: {
-    id: Id;
-    date: string;
-    coaching_relationship_id: Id;
-    created_at: DateTime;
-    updated_at: DateTime;
-  };
+  coachingSession: CoachingSessionType;
+  onUpdate: () => void;
 }
 
 const CoachingSession: React.FC<CoachingSessionProps> = ({
   coachingSession,
+  onUpdate,
 }) => {
   const { setCurrentCoachingSessionId } = useCoachingSessionStateStore(
     (state) => state
   );
-  const [updateDialogOpen, setUpdateDialogOpen] = useState(false);
 
   return (
-    <>
-      <Card>
-        <CardHeader className="p-4">
-          <div className="flex flex-col sm:flex-row sm:items-center justify-between gap-2">
-            <div className="space-y-1">
-              <OverarchingGoal coachingSessionId={coachingSession.id} />
-              <div className="text-sm text-muted-foreground">
-                {format(new Date(coachingSession.date), "MMMM d, yyyy h:mm a")}
-              </div>
+    <Card>
+      <CardHeader className="p-4">
+        <div className="flex flex-col sm:flex-row sm:items-center justify-between gap-2">
+          <div className="space-y-1">
+            <OverarchingGoal coachingSessionId={coachingSession.id} />
+            <div className="text-sm text-muted-foreground">
+              {format(new Date(coachingSession.date), "MMMM d, yyyy h:mm a")}
             </div>
+          </div>
+          <div className="flex items-center gap-2">
+            <Link href={`/coaching-sessions/${coachingSession.id}`} passHref>
+              <Button
+                size="sm"
+                className="w-full sm:w-auto mt-2 sm:mt-0 text-sm px-3 py-1"
+                onClick={() => setCurrentCoachingSessionId(coachingSession.id)}
+              >
+                Join Session
+              </Button>
+            </Link>
             <DropdownMenu>
               <DropdownMenuTrigger asChild>
                 <Button variant="ghost" size="icon">
                   <MoreHorizontal className="h-4 w-4" />
                 </Button>
               </DropdownMenuTrigger>
               <DropdownMenuContent align="end">
-                <DropdownMenuItem asChild>
-                  <Link
-                    href={`/coaching-sessions/${coachingSession.id}`}
-                    onClick={() =>
-                      setCurrentCoachingSessionId(coachingSession.id)
-                    }
-                  >
-                    Join Session
-                  </Link>
-                </DropdownMenuItem>
-                <DropdownMenuItem onClick={() => setUpdateDialogOpen(true)}>
+                <DropdownMenuItem onClick={onUpdate}>
```

Comment:
To be consistent with Agreements and Actions, can you update the drop down menu item text to be "Edit" and "Delete" instead of "Update Session" and "Delete Session"?

Reply:
Updated 536a763ba3acddbbfb02dc1dfae19c24dc55447b